backend/experiment/models.py

```diff
@@ -44,6 +44,35 @@ def consent_upload_path(instance, filename):
     return 'consent/{0}/{1}'.format(folder_name, filename)
 
 
+class ExperimentSeriesGroup(models.Model):
+    name = models.CharField(max_length=64, blank=True, default='')
+    series = models.ForeignKey(ExperimentSeries, on_delete=models.CASCADE)
+    order = models.IntegerField(default=0)
```

Comment:
I think we should impose a [unique_together](https://docs.djangoproject.com/en/5.0/ref/models/options/#unique-together)  constraint on order / name / series, such that we cannot have two `ExperimentSeriesGroup`s with the same order integer.

Reply:
In principle I do agree, but supporting this is hard. You will probably run into problems as the items will be saved individually, so if you change the order of two groups (from GA 0 and GB 1 to GA 1 and GB 0), the db will complain that there is already an group (GB) with order number 1 when you try to save GA with number 1. So then you will have to do magic things in order to facilitate it like first saving GB temporarily with a very high or negative order number like 999 or -1, then save GA with order number 1, and then save GB with order number 0. Perhaps it would be enough to just warn the user in the admin interface? I mean, technically it doesn't break the application when two groups have the same order number, but you might get random orders depending on the request.

Reply:
Hm, that is a good point. In that case, consider the "changes requested" to be adding the help_text.